### PR TITLE
Allow named polls

### PR DESCRIPTION
**Pull request by tojofo**
This PR introduces the ability to name polls.  This affords multiple polls against the same resource for example `GET /conversations` can be polled less frequently than `GET /conversations/99`

@mfpiccolo this is my first contribution to an individual's project.  Please feel free to tell me what you want me to do to make the merge as easy possible for you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ var ApplicationRoute = Ember.Route.extend({
   afterModel: function () {
     this._super(...arguments);
     this.get('poll').start({
-      idle_timeout: 10000,
+      idleTimeout: 10000,
```
Comment (mfpiccolo) [inline README.md]: This should be back to underscore case?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #  ember-cli-poll
 
-This is an npm package that contains a polling serivce for ember-data
+This is a fork of an npm package that contains a polling serivce for ember-data
```
Comment (mfpiccolo) [inline README.md]: Revert this change

File: README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ packaged as an [Ember CLI](https://github.com/stefanpenner/ember-cli) Addon.
 To install simply run
 
 ```
-ember install ember-cli-poll
+ember install https://github.com/Biteable/ember-cli-poll.git
```
Comment (mfpiccolo) [inline README.md]: Revert this change

**Comment by mfpiccolo** [comment]
@tojofo Thanks for putting this together and the change log as well.  If you could fix the few comments I can merge and push a new version to npm.

**Comment by tojofo** [comment]
Nice find re `idle_timeout`.  Changes made and pushed.

**Comment by mfpiccolo** [comment]
Thanks @tojofo!  v0.1.0 is released.

**Comment by tojofo** [comment]
Excellent, thanks for helping me contribute
